CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## New Features
 
+- PR #697: Silhouette Score metric ml-prim
```

Comment:
I think you're missing a merge of branch-0.9 here. That's why this line currently comes under branch-0.8